content/about/manager_architecture/metrics-flow.md

```diff
@@ -7,6 +7,8 @@ abstract: Describes the flow of streaming metrics from a host to Cloudify's Mana
 weight: 700
 aliases: /manager_architecture/metrics-flow/
 ---
+Note that Riemann, InfluxDB, and the proprietary poller (amqpinflux) are no longer installed by default, allowing for slimmer, cleaner managers where these capabilities are not required.
```

Comment:
{{% note %}} For Cloudify Manager version 4.4 and above, Riemann, InfluxDB, and the proprietary poller (amqpinflux) are not installed by default to reduce storage requirements and improve ease of maintenance where these capabilities are not required. {{% /note %}}

(What do you mean by "where these capabilities are not required"?)